src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedNamedTypeSymbol.cs

```diff
@@ -123,6 +124,10 @@ public override ImmutableArray<Location> Locations
         {
             get
             {
+                if (IsTupleType)
+                {
+                    return TupleData.Locations;
```

Comment:
>Locations [](start = 37, length = 9)

I assume this is an improvement in the experience.  Nit: add an empty line after the `}`, and in the next method too.

Reply:
Yes, we need to keep locations pointing to where various tuples and tuple element names are defined in source.
Without this, GoToDefinition and FindAllReferences works badly.

---
In reply to: [357376280](https://github.com/dotnet/roslyn/pull/39370#discussion_r357376280) [](ancestors = 357376280)